mitosheet/mitosheet/public/v3/sheet_functions/misc_functions.py

```diff
@@ -149,11 +150,59 @@ def GETNEXTVALUE(series: pd.Series, condition: BoolRestrictedInputType) -> pd.Se
 
     return GETPREVIOUSVALUE(reversed_series, reversed_condition)[::-1]
 
+@cast_values_in_arg_to_type('index', 'int')
+def VLOOKUP(lookup_value: AnyPrimitiveOrSeriesInputType, where: pd.DataFrame, index: IntRestrictedInputType) -> pd.Series:
+    """
+    {
+        "function": "VLOOKUP",
+        "description": "Returns the next value from series that meets the condition.",
+        "search_terms": ["vlookup", "merge", "join", "search", "lookup"],
+        "examples": [
+            "VLOOKUP(Names0, Sample_Names:Sample_Ages, 1, False)",
```

Comment:
Maybe we could pick more informative sample values. Like `ids` as the columns or something. And the where is `ids:age` or something. Sample is kinda a filler word.

Reply:
How does that updated version look?